staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -41,15 +43,21 @@ import (
 	"k8s.io/utils/trace"
 )
 
+const defaultExpectedTypeName = "<unspecified>"
+
 // Reflector watches a specified resource and causes all changes to be reflected in the given store.
 type Reflector struct {
 	// name identifies this reflector. By default it will be a file:line if possible.
 	name string
 	// metrics tracks basic metric information about the reflector
 	metrics *reflectorMetrics
 
-	// The type of object we expect to place in the store.
-	expectedType reflect.Type
+	// The name of the type we expect to place in the store.
```

Comment:
clarify this is a stringification of expectedUnstructuredGVK if that is provided, and a stringification of expectedType otherwise

Reply:
Done